egs/babel_multilang/s5/local/nnet3/run_common_langs.sh

```diff
@@ -0,0 +1,121 @@
+      fi
+      steps/compute_cmvn_stats.sh data/$L/${datadir}_tmp exp/$L/make_plp/${datadir}_tmp $featdir || exit 1;
+      utils/fix_data_dir.sh data/$L/${datadir}_tmp
+      
+      utils/copy_data_dir.sh --spk-prefix sp1.0- --utt-prefix sp1.0- data/$L/${datadir} data/$L/temp0
+      utils/combine_data.sh data/$L/${datadir}_sp data/$L/${datadir}_tmp data/$L/temp0
+      utils/fix_data_dir.sh data/$L/${datadir}_sp
+      rm -r data/$L/temp0 data/$L/${datadir}_tmp
+    done
+  fi
+  
+  train_set=train_sp
+  if [ $stage -le 2 ] && [ "$generate_alignments" == "true" ] && [ ! -f exp/$L/tri5_ali_sp/.done ]; then
+    #obtain the alignment of the perturbed data
+    steps/align_fmllr.sh \
+      --nj 70 --cmd "$train_cmd" \
+      --boost-silence $boost_sil \
+      data/$L/$train_set data/$L/lang exp/$L/tri5 exp/$L/tri5_ali_sp || exit 1
+    touch exp/$L/tri5_ali_sp/.done
+  fi
+fi
+
+if [ $stage -le 3 ] && [ ! -f data/$L/${train_set}_hires/.done ]; then
+  mfccdir=mfcc_hires/$L
+  if [[ $(hostname -f) == *.clsp.jhu.edu ]] && [ ! -d $mfccdir/storage ]; then
+    date=$(date +'%m_%d_%H_%M')
+    utils/create_split_dir.pl /export/b0{1,2,3,4}/$USER/kaldi-data/egs/$L-$date/s5c/$mfccdir/storage $mfccdir/storage
+  fi
+
+  # the 100k_nodup directory is copied seperately, as
+  # we want to use exp/tri2_ali_100k_nodup for lda_mllt training
+  # the main train directory might be speed_perturbed
+  for dataset in $train_set ; do
+    utils/copy_data_dir.sh data/$L/$dataset data/$L/${dataset}_hires
+
+    # scale the waveforms, this is useful as we don't use CMVN
+    data_dir=data/$L/${dataset}_hires
+
+    utils/data/perturb_data_dir_volume.sh $data_dir || exit 1 ; 
+
+    steps/make_mfcc.sh --nj 70 --mfcc-config conf/mfcc_hires.conf \
+        --cmd "$train_cmd" data/$L/${dataset}_hires exp/$L/make_hires/$dataset $mfccdir;
+    steps/compute_cmvn_stats.sh data/$L/${dataset}_hires exp/$L/make_hires/${dataset} $mfccdir;
+
+    # Remove the small number of utterances that couldn't be extracted for some
+    # reason (e.g. too short; no such file).
+    utils/fix_data_dir.sh data/$L/${dataset}_hires;
+  done
+  touch data/$L/${train_set}_hires/.done
+fi
+
+if [ $stage -le 4 ]; then
+  if [[ "$use_pitch" == "true" ]]; then
+    echo use_pitch = $use_pitch
+    pitchdir=pitch/$L
+    train_set=${train_set}_hires
+    for dataset in $train_set; do
+      if $use_pitch; then
+        mkdir -p $pitchdir
+        if [ ! -f data/$L/${dataset}_pitch/feats.scp ]; then
+        utils/copy_data_dir.sh data/$L/$dataset data/$L/${dataset}_pitch
+        steps/make_pitch.sh --nj 70 --pitch-config $pitch_conf \
+          --cmd "$train_cmd" data/$L/${dataset}_pitch exp/$L/make_pitch/${dataset} $pitchdir;
+        fi
+        aux_suffix=_pitch
+      fi
+
+      if [ ! -f data/$L/${dataset}_mfcc${aux_suffix}/feats.scp ]; then
+        steps/append_feats.sh --nj 16 --cmd "$train_cmd" data/$L/${dataset} \
```

Comment:
I agree with Vijay that it would be a good idea to extend it in the same way-- now is a good opportunity to test the changes, since you'll have to rerun your stuff to test it after other changes anyway.